copier.yml

```diff
@@ -26,12 +26,33 @@ schedule_interval:
   help: Cron expression
   default: 0 12 * * *
 
+use_ingestion:
+    type: bool
+    help: Would you like to use ingestion framework?
+    default: false
+
+ingestion_framework:
+    type: str
+    when: "{% if use_ingestion %}true{% endif %}"
+    help: What ingestion framework would you like to use?
+    choices:
+        Airbyte: airbyte
+    default: airbyte
+
+destination_id:
+    type: str
+    when: "{% if ingestion_framework == 'airbyte' %}true{% endif %}"
+    help: Destination Id 
+
+source_id:
+    type: str
+    when: "{% if ingestion_framework == 'airbyte' %}true{% endif %}"
+    help: Source Id
+
 _exclude:
   - .git
   - .github
 
-_min_copier_version: "5.1.0"
```

Comment:
Removed due to not having the possibility to use conditional statements with files.

It's a good idea to pin the actual version, `6.1.0` ?

Reply:
Already been tested and with pinned version works fine.

Reply:
The question is if we can use 6.1.0 in dp tool (dependencies)? If yes that's fine to pin.

Reply:
No, in dp tool `5.1.0` is used, but while testing on version `5.1.0` conditional statement didn't work

Reply:
but have you upgraded it in dp?

Reply:
I will bump version to `6.1.0` in dp